governance/SIG-contributing-template.md

```diff
@@ -0,0 +1,50 @@
+## We welcome contribution
+Describe what kind of issues a new contributor could pick.
+
+[TODO:] A list of standardized labels “good first issue”?, “contribution welcome”?, etc.
+
+## Contributor checklist
+* Comment on an existing ticket or open a new one before coding any PR
+* Wait for a specific approval reply from mantainers and an `approved-for-PR` label
+* Read and Sign the CLA
+* Follow Tensorflow coding style
+* Follow extra repository specific code style
+* Write and execute tests
+* Lint your code
+
+## Environment setup
+How to checkout the code and mount the volume inside a container
+
+
+## Build from source
+How to build the code from source inside a container
+
+## Codestyle 
+Link to Google style guides and any other specific code style
+Linting info and how to configure pre-commit hooks
+
+## IDE
+Any available info for working with specific IDE
+How to use [Codespaces/Devcontainers](https://github.com/microsoft/vscode-dev-containers) if supported
+
+## DOCS
+To contribute to the documentation please follow [DOCS.md]
+
+## TESTS
+How to write and execute tests (bazel, pytest, etc.) or a link to a specific [TEST.md]
+
+## DEBUG
+How to debug your code
+
+## CI
+Quickly describe your CI
+
+## CODEOWNERS
```

Comment:
It  could be great if we could standardize and automatically [monitor the codeowners/mentors](https://github.com/tensorflow/addons/pull/2024) availability.

Reply:
We could take some idea from the [Addons proxy maintainership](https://github.com/tensorflow/community/blob/master/rfcs/20190308-addons-proxy-maintainership.md) also if some of these theoretic best practices are not applied.